<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>671</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Asiakas Hallinta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>570</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Poista</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGet">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Päivitä</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Etsi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddCustomer">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>560</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lisää asiakas</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>561</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="customerinfo">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>421</width>
        <height>621</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tiedot</string>
       </attribute>
       <widget class="QLineEdit" name="lineEditMedication">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>480</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditEmail">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>250</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditFirstname">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Etunimi:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAddress">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>170</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>440</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sairaudet:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sukunimi:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>480</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Lääkitys:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Harrastukset:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditProfession">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>360</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSocialsecurity">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Yhteystiedot:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Hetu:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditLastname">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditDiseases">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>440</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>530</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Tallenna</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Osoite:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditPhone">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>210</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Puhnro:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>530</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Kumoa</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditHobbies">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>400</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Esitiedot:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ammatti:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Käynnit</string>
       </attribute>
       <widget class="QPushButton" name="addVisitButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Lisää käynti</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>281</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>50</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Muokkaa käyntejä</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEditFirstname_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>410</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Lääkkeet</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Puhnro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>510</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Kumoa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Osoite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>450</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Harrastukset</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sukunimi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Etunimi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sairaudet</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditDiseases_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>370</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMedication_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>410</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSocialsecurity_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Hetu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>510</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Tallenna</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditHobbies_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>450</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProfession_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ammatti</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPhone_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEmail_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLastname_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAddress_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>521</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>          Lisää uusi asiakas</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>         Asiakkaat</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
